dojo/tools/awssecurityhub/parser.py

```diff
@@ -36,41 +34,27 @@ def get_items(self, tree, test):
 
 
 def get_item(finding, test):
+    finding_id = finding.get('Id', "").split('/')[-1]
     title = finding.get('Title', "")
     severity = finding.get('Severity', {}).get('Label', 'INFORMATIONAL').title()
     description = finding.get('Description', "")
+    resources = finding.get('Resources', "")
+    resource_id = resources[0]['Id'].split(':')[-1]
     mitigation = finding.get('Remediation', {}).get('Recommendation', {}).get('Text', "")
     references = finding.get('Remediation', {}).get('Recommendation', {}).get('Url')
-    cve = None
-    cwe = None
     false_p = False
-    duplicate = False
-    out_of_scope = False
-    impact = None
-
-    if finding.get('Compliance', {}).get('Status', "PASSED"):
-        if finding.get('LastObservedAt', None):
-            try:
-                mitigated = datetime.strptime(finding.get('LastObservedAt'), "%Y-%m-%dT%H:%M:%S.%fZ")
-            except:
-                mitigated = datetime.strptime(finding.get('LastObservedAt'), "%Y-%m-%dT%H:%M:%fZ")
-        else:
-            mitigated = datetime.utcnow()
-    else:
```

Comment:
can you explain how this competes with close old findings? I think PASSED findings are still imported so will not be closed by close_old_findings?

Reply:
I've re-instated the code above. In my team's implementation/workflow we will not hit this path since we're using boto to filter the results and only report on new findings but the proposed change may negatively impact other implementations.